### PR TITLE
Adds intro to Wormhole SDK page

### PR DESCRIPTION
**Pull request by dawnkelly09**
### Description

1. Adds Introduction paragraph 
2. Builds out copy a bit in sections
3. Added cards with links to look like other landing type pages
4. Added a few things from my expanded SDK section reference page (tried not to do too much)

### Checklist

- [ x] **Required** - I have added a label to this PR 🏷️
- [ x] **Required** - I have run my changes through Grammarly
- [ ] If pages have been moved, I have created redirects in the `wormhole-mkdocs` repo

### CODE REVIEW

File: build/applications/wormhole-sdk.md [hunk 1]
```diff
@@ -193,7 +327,7 @@ The payload contains the information necessary to perform whatever action is req
 
 #### Token Bridge
 
-The most familiar protocol built on Wormhole is the Token Bridge. Every chain has a `TokenBridge` protocol client that provides a consistent interface for interacting with the Token Bridge. This includes methods to generate the transactions required to transfer tokens and methods to generate and redeem attestations. `WormholeTransfer` abstractions are the recommended way to interact with these protocols but it is possible to use them directly.
+The most familiar protocol built on Wormhole is the Token Bridge. Every chain has a `TokenBridge` protocol client that provides a consistent interface for interacting with the Token Bridge which includes methods to generate the transactions required to transfer tokens and methods to generate and redeem attestations. `WormholeTransfer` abstractions are the recommended way to interact with these protocols but it is possible to use them directly.
```
Comment (eshaben) [inline build/applications/wormhole-sdk.md:330]:
```suggestion
The most familiar protocol built on Wormhole is the Token Bridge. Every chain has a `TokenBridge` protocol client that provides a consistent interface for interacting with the Token Bridge, which includes methods to generate the transactions required to transfer tokens and methods to generate and redeem attestations. `WormholeTransfer` abstractions are the recommended way to interact with these protocols, but it is possible to use them directly.
```

**Review by eshaben — COMMENTED** [review]
Sweet! Thanks!

File: build/applications/wormhole-sdk.md [hunk 2]
```diff
@@ -97,39 +142,127 @@ Optionally, you can override the default configuration with a partial `WormholeC
...
 Wormhole currently supports the following platforms:
 
-- EVM
-- Solana
-- Cosmwasm
-- Sui
-- Aptos
+??? interface "EVM"
+
+    ??? child "Available protocols"
```
[the remark this answers is not in the document]
Comment (dawnkelly09) [reply build/applications/wormhole-sdk.md:162]: Removed the child admonition. Now it's just a list under each platform (one drop)

File: build/applications/wormhole-sdk.md [hunk 3]
```diff
@@ -97,39 +142,127 @@ Optionally, you can override the default configuration with a partial `WormholeC
...
+
 
 See the [Platforms folder of the TypeScript SDK](https://github.com/wormhole-foundation/wormhole-sdk-ts/tree/main/platforms){target=\_blank} for an up-to-date list of the platforms supported by the Wormhole TypeScript SDK.
 
 ### Chain Context
 
-The `Wormhole` class provides a `getChain` method that returns a `ChainContext` object for a given chain. This object provides access to the chain specific methods and utilities. Much of the functionality in the `ChainContext` is provided by the `Platform` methods but the specific chain may have overridden methods.
+The `definitions` package of the SDK includes the `ChainContext` class, which creates an interface for working with connected chains in a standardized way. This class contains the network, chain, and platform configurations for connected chains and cached RPC and protocol clients. The `ChainContext` class also exposes chain-specific methods and utilities. Much of the functionality comes from the `Platform` methods but some specific chains may have overridden methods via the context.
+
+The `ChainContext` class defines the following key type parameters which are used throughout the package:
```
[the remark this answers is not in the document]
Comment (dawnkelly09) [reply build/applications/wormhole-sdk.md:244]: Took the parameters admonition out and just added a sentence about this is where the chain, platform, and network type parameters are defined.

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**